aws_lambda_builders/workflows/java_maven/workflow.py

```diff
@@ -0,0 +1,58 @@
+"""
+Java Maven Workflow
+"""
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from aws_lambda_builders.workflows.java_gradle.utils import OSUtils
```

Comment:
We don't want to be importing other workflows modules. Even if it is the same, please copy into a maven specific OSUtils. We know there is reuse across workflows and want to solve this all at once. If workflows start importing other workflow classes, it will make it difficult to untangle the imports. Each workflow should be isolated from each other.